.github/workflows/android-build-check.yml

```
@@ -0,0 +1,189 @@
+name: Android Build Check
+
+on:
+  push:
+    branches: ["**"]
+    paths:
+      - "apps/mobile/**"
+      - "package.json"
+      - "pnpm-workspace.yaml"
+      - "pnpm-lock.yaml"
+<<<<<<< HEAD
```

actionlint (1.7.10)

could not parse as YAML: could not find expected ':'

(syntax-check)